content/data/data.md

```diff
@@ -0,0 +1,43 @@
+---
+date: "10 June 2019"
+title: "AnVIL Data"
+author: "AnVIL"
+---
+
+
+# Data 
+
+
+[[heroSmall]]
+| The AnVIL hosts high value datasets relevant to human health and disease.
+
+
+ AnVIL data sets include NHGRI funded data sets of the Centers for Commons Disease Genetics (CCDG), the Centers for Mendelian Genomics (CMG), and the Genotype Tissue-Expression project (GTEx). Additional data sets include the high coverage 1000 Genomes whole genome sequencing data and VCFs, with more data sets to be added over time. 
```

Comment:
"Centers for Common Disease Genetics..." (not plural on Common)

add links? 
https://www.genome.gov/Funded-Programs-Projects/NHGRI-Genome-Sequencing-Program/Centers-for-Common-Disease-Genomics
http://mendelian.org/
https://gtexportal.org/home/
https://www.internationalgenome.org/

Reply:
oh duh, I see the links below now! nice!